Sustainable sourcing layers

tylere opened the pull request:
Updates the **Sustainable Sourcing Layers 2025a visualization** notebook.

Summary of changes:
- Moves user information (project & region) to Colab secrets
- Fixes dictionary set error
- Updates comments and variable naming for readability

tylere committed:
Add Sustainable Sourcing Layers from Colab

tylere committed:
Minor change (print EE version)

tylere committed:
Removed cell outputs.

tylere committed:
Print geemap version

tylere committed:
Update the authentication to use Colab secrets

tylere committed:
Reformat addLayer() calls to improve readability

tylere committed:
Describe the authentication workflow.

tylere committed:
Fix dictionary set error. Rename to natural_forest2020 for consistency.

tylere committed:
Add missing import (pprint)

tylere committed:
After clearing outputs

tylere committed:
Convert thresholds from list to dict.

tylere committed:
Add comments. Rename variables.

tylere committed:
Spelling fixes.

tylere committed:
Fix spelling error

tylere committed:
After clearing outputs

brycecr commented on notebooks/PUBLIC_Sustainable_Sourcing_Layers_2025a.ipynb:
```diff
@@ -0,0 +1,604 @@
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Google Deep Mind Natural Forest\n",
```
nit: DeepMind is one word

tylere replied:
Fixed by [1aa38f0](https://github.com/google/forest-data-partnership/pull/12/commits/1aa38f0e24776d80b0af9b9c8ffd2d09f11d7a17)

brycecr commented on notebooks/PUBLIC_Sustainable_Sourcing_Layers_2025a.ipynb:
```diff
@@ -0,0 +1,604 @@
+      "source": [
+        "## Google Deep Mind Natural Forest\n",
+        "\n",
+        "Create a variable that references the natural forest dataset described in [Neumann et al. 2025](https://eartharxiv.org/repository/view/9085/)."
```
might additionally be nice to link the Nature Trace catalog as here as you linked the FDaP catalog https://developers.google.com/earth-engine/datasets/publisher/nature-trace

tylere replied:
Addressed by [b412ba2](https://github.com/google/forest-data-partnership/pull/12/commits/b412ba2c1d8fe580fcfb59594c3deccfc4450940)

brycecr commented on notebooks/PUBLIC_Sustainable_Sourcing_Layers_2025a.ipynb:
```diff
@@ -0,0 +1,604 @@
+        "\n",
+        "For details on how these layers were produced, see [the technical documentation on GitHub](https://github.com/google/forest-data-partnership/tree/main/models).  In particular, see [the limitations](https://github.com/google/forest-data-partnership/tree/main/models#limitations).  See also the [Forest Data Partnership publisher catalog](https://developers.google.com/earth-engine/datasets/p ...
+        "\n",
+        "Note that users of commercial projects will need to request access to the Forest Data Partnership datasets with [this form](https://docs.google.com/forms/d/e/1FAIpQLSe7L3eh6t2JIPqEtAQwXwY7ZmW52v8W5vrIi4QN_XYgTNJZLw/viewform).\n",
```
please remove this line; we no longer require an exemption for commercial users

tylere replied:
Addressed by [553fa12](https://github.com/google/forest-data-partnership/pull/12/commits/553fa12469f0b021bfb29d2f2479d46705b26574)

brycecr commented on notebooks/PUBLIC_Sustainable_Sourcing_Layers_Cloud_Function.ipynb:
```diff
@@ -0,0 +1,867 @@
+        "\n",
+        "This notebook demonstrates deployment of Google Cloud functions to get commodity and forest information in a format designed to be integrated to other workflows.\n",
+        "\n",
+        "The 2025a release of Forest Data Partnership sustainable sourcing layers including palm, rubber, cocoa and coffee are used for commodity information.  For details on how these layers were produced, see [the technical documentation on GitHub](https://github.com/google/forest-data-partnership/tree/main/models).  In particular, see [the limitations](https://github.com/google/forest-data-par ...
```
nit:
Possible drop-in for first sentence

The Forest Data Partnership sustainable sourcing layers provide per-pixel probability estimates for plantation presence. The 2025a release covers palm, rubber, cocoa, and coffee.

tylere replied:
Revised text in [c09b21c](https://github.com/google/forest-data-partnership/pull/12/commits/c09b21cdb834eef958e4adaba0c43327f2f8b4a8)

brycecr commented on notebooks/PUBLIC_Sustainable_Sourcing_Layers_Cloud_Function.ipynb:
```diff
@@ -0,0 +1,867 @@
+        "\n",
+        "The 2025a release of Forest Data Partnership sustainable sourcing layers including palm, rubber, cocoa and coffee are used for commodity information.  For details on how these layers were produced, see [the technical documentation on GitHub](https://github.com/google/forest-data-partnership/tree/main/models).  In particular, see [the limitations](https://github.com/google/forest-data-par ...
+        "\n",
+        "Note that users of commercial projects will need to request access through [this form](https://docs.google.com/forms/d/e/1FAIpQLSe7L3eh6t2JIPqEtAQwXwY7ZmW52v8W5vrIi4QN_XYgTNJZLw/viewform).\n",
```
no longer true, please remove

tylere replied:
Removed in [7789636](https://github.com/google/forest-data-partnership/pull/12/commits/77896361260cd2e15fa632b249f847bb1f4c7339)

brycecr commented on notebooks/PUBLIC_Sustainable_Sourcing_Layers_Cloud_Function.ipynb:
```diff
@@ -0,0 +1,867 @@
+      "cell_type": "code",
+      "source": [
+        "# USE YOUR OWN PROJECT!\n",
+        "PROJECT = 'forest-data-partnership'\n",
```
please replace with a different placeholder (perhaps one that is more clearly a placeholder); could we consider setting as a var for easy replacement through the script? (could maybe make it an input box too)

brycecr replied:
Please update to a different placeholder

brycecr commented on notebooks/PUBLIC_Sustainable_Sourcing_Layers_Cloud_Function.ipynb:
```diff
@@ -0,0 +1,867 @@
+    {
+      "cell_type": "code",
+      "source": [
+        "import ee\n",
```
nit: the early import cell already initialized

brycecr reviewed (changes requested):
mostly small comments. not all need to be addressed

brycecr reviewed (commented):
also was curious why these file names have PUBLIC_ on them? If they're in our notebooks folder that's somewhat implied, no?

tylere commented:
> also was curious why these file names have PUBLIC_ on them? If they're in our notebooks folder that's somewhat implied, no?

The original sources were notebooks stored on Google Drive, which had names that started with "PUBLIC ". I agree that the prefix no longer seems needed, so I will remove it.

tylere committed:
Link to Nature Trace collection

tylere committed:
Fix DeepMind spelling

tylere committed:
Remove note on commercial access

tylere committed:
Improve Forest Data Partnership sustainable sourcing layers description

tylere committed:
Remove commercial access note

tylere committed:
Removing addition of PUBLIC_Sustainable_Sourcing_Layers_Cloud_Function.ipynb

The content of that notebook is addressed in
https://github.com/google/forest-data-partnership/pull/14

tylere committed:
Rename notebook (remove PUBLIC_)

n-clinton reviewed (approved):
LGTM
